src/cephadm/cephadm

```diff
@@ -1351,7 +1352,7 @@ def check_units(units, enabler=None):
 
 def get_legacy_config_fsid(cluster, legacy_dir=None):
     # type: (str, str) -> Optional[str]
-    config_file = '/etc/ceph/%s.conf' % cluster
+    config_file = ETC_CEPH_DIR + '/%s.conf' % cluster
```

Comment:
note, this is almost always SHELL_DEFAULT_CONF, but has nothing to do with SHELL_DEFAULT_CONF

Reply:
I think this one is good as it is? ... since we need to support legacy configs that contain the cluster name ...